test/corpus/errors.txt

```diff
@@ -141,3 +141,45 @@ $c$;
           (UNEXPECTED 'c')
           (UNEXPECTED ';'))
         (MISSING dollar_quote)))))
+
+================================================================================
+Function body with a tag name contains whitespace
+================================================================================
+
+create or replace function public.do_stuff()
+  returns trigger
+  language plpgsql
+as $a $
+begin
+  return $b$text$b$;
+end;
+$a $;
+
+--------------------------------------------------------------------------------
+
+(program
+  (ERROR
+    (keyword_create)
+    (keyword_or)
+    (keyword_replace)
+    (keyword_function)
+    (object_reference
+      (identifier)
+      (identifier))
+    (function_arguments)
+    (keyword_returns)
+    (keyword_trigger)
+    (function_language
+      (keyword_language)
+      (identifier))
+    (ERROR
+      (keyword_as)
+      (UNEXPECTED 'a')
+      (UNEXPECTED '\n'))
```

Comment:
well, this will not work on windows machines. 
any ideas @DerekStride ?

Reply:
Could the test be written as `as $a $ begin` ?

Reply:
I tried `as $a $ begin` but it produced strange behavior with `(UNEXPECTED ')`, after `test --update` test just fails with diff which doesn't contains expected/actual.
Changing `$b $text$b $; ` doesn't capture any newlines and works.